src/posix/platform/radio.cpp

```diff
@@ -561,11 +567,17 @@ otError otPlatDiagProcess(otInstance *aInstance, uint8_t aArgsLength, char *aArg
     }
 #endif
 
+    if (strcmp(aArgs[0], "radiospinel") == 0)
+    {
+        return GetRadioSpinel().RadioSpinelDiagProcess(aArgs, aArgsLength);
```

Comment:
I know it may be easier to use `return` here, but suggest changing to use `ExitNow()` so we can follow the "single return from any function/method" policy from OT style guide.

Reply:
oh yes, I was planning to rework this function, but it totally slipped my mind... thanks for spotting that!